docs/flink-getting-started.md

```diff
@@ -860,16 +860,17 @@ INSERT INTO tableName /*+ OPTIONS('upsert-enabled'='true') */
-| distribution-mode      | Table write.distribution-mode              | Overrides this table's write.distribution-mode                                                             |
-| compression-codec      | Table write.(fileformat).compression-codec | Overrides this table's compression codec for this write                                                    |
-| compression-level      | Table write.(fileformat).compression-level | Overrides this table's compression level for Parquet and Avro tables for this write                        |
-| compression-strategy   | Table write.orc.compression-strategy       | Overrides this table's compression strategy for ORC tables for this write                                  |
+| Flink option           | Default                                    | Description                                                                                                                        |
+|------------------------|--------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|
+| write-format           | Table write.format.default                 | File format to use for this write operation; parquet, avro, or orc                                                                 |
+| target-file-size-bytes | As per table property                      | Overrides this table's write.target-file-size-bytes                                                                                |
+| upsert-enabled         | Table write.upsert.enabled                 | Overrides this table's write.upsert.enabled                                                                                        |
+| overwrite-enabled      | false                                      | Overwrite the table's data, overwrite mode shouldn't be enable when configuring to use UPSERT data stream.                         |
+| distribution-mode      | Table write.distribution-mode              | Overrides this table's write.distribution-mode                                                                                     |
+| compression-codec      | Table write.(fileformat).compression-codec | Overrides this table's compression codec for this write                                                                            |
+| compression-level      | Table write.(fileformat).compression-level | Overrides this table's compression level for Parquet and Avro tables for this write                                                |
+| compression-strategy   | Table write.orc.compression-strategy       | Overrides this table's compression strategy for ORC tables for this write                                                          |
+| write-parallelism      | null                                       | Configuring the write parallel number for iceberg stream writer. By default, it is the same as the parallel of the input operator. |
```

Comment:
Nit: I generally feel we should avoid coding specific terms like `null` in these kinds of docs (it makes sense for specs or data types because `null` is a necessary concept). The default just needs to describe the default value or behavior in this case imo. Since the default is described as `the same as the parallel of the input operator to the write operator.` could we just put that in instead of null?

Reply:
Reasonable, modified. Thx.